src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneNodeFormula.kt

```diff
@@ -52,7 +52,8 @@ internal class StandaloneNodeFormula(
                 jiraIp = ssh.host.ipAddress
             )
             connection.execute("echo jira.home=`realpath $jiraHome` > $unpackedProduct/atlassian-jira/WEB-INF/classes/jira-application.properties")
-            connection.execute("echo jira.autoexport=false > $jiraHome/jira-config.properties")
+            connection.execute("touch $jiraHome/jira-config.properties")
+            connection.execute("grep -qxF \"jira.autoexport=false\" $jiraHome/jira-config.properties || echo -e \"\\njira.autoexport=false\" >> $jiraHome/jira-config.properties")
```

Comment:
It looks error prone. What if we add `jira.autoexport = false`. It will still work, but will not be detected by grep.
It's still better than before, so I would not block the review 👍 


We may also not need the property. I think it doesn't work like we expected (https://ecosystem.atlassian.net/browse/JPERF-344).

Reply:
And AFAIK all our datasets already have the property set :)

Reply:
I think we could improve the grep, but I am going away till the end of Apr, so could do that in 2 weeks time. And yes, all our datasets have that already, my problem was that the jira-config.properties file was always overwritten, so I changed this to be less invasive. But if we can get rid of this altogether through the issue you linked, then I would be just as happy.

Reply:
👍  Just add the issue key and changelog and we can merge. (Remember to rebase!)

Reply:
If this fixes JPERF-344, I wonder if the integration tests will become faster here. I hope we see it in the CI results.

Reply:
> If this fixes JPERF-344, I wonder if the integration tests will become faster here. I hope we see it in the CI results.

It doesn't fix JPERF-344. AFAIK we see the export even if the flag is set.